src/__tests__/shell/secret.sh

```diff
@@ -0,0 +1 @@
+echo "Hello $NODE_ENV"
```

Comment:
Does this need a newline at the end? I never know when that will and won't potentially break things.

Reply:
Good question... I _think_ the `echo` will automatically add one.

Reply:
Oh! I totally misread this comment. You meant: does the *file* need a newline... not the `echo` statement. The answer is that it should have a new line and I totally missed it.